src/options.zig

```diff
@@ -2588,7 +2588,13 @@ pub const PathTemplate = struct {
     }
 
     pub fn format(self: PathTemplate, comptime _: []const u8, _: std.fmt.FormatOptions, writer: anytype) !void {
-        var remain = self.data;
+        var pathbuf: [std.fs.MAX_PATH_BYTES]u8 = undefined;
+        var remain = pathbuf[0..self.data.len];
+        @memcpy(remain, self.data);
+        for (remain) |*c| {
```

Comment:
Can you use your posixToPlatformInPlace function?

Reply:
ah yep, wrote that block before I added it and forgot to come back